docs/diagrams/sep6/withdraw.md

```diff
@@ -0,0 +1,47 @@
+### Withdraw
+
+This diagram illustrates the withdraw flow. The flow starts after the user has authenticated with the anchor via SEP-10 and provided some basic information about themselves. The example here shows the user withdrawing funds to their bank account, but the flow is similar for other withdrawal methods. The `withdraw-exchange` flow works similarly, but the Platform will additionally verify the quote requested or make a call to the Fee integration to update amounts.
+
```

Comment:
Maybe we should remove the "and provided basic information about themselves" here since the diagram shows the user providing KYC information as a result of initiating a transaction.

Reply:
I'll make the same change in the `deposit` PR as well.